.ci/azure-pipelines/build_windows.bat

```diff
@@ -23,17 +23,26 @@
 ::
 :: Defined in .azure-pipelines.yml:
 ::
-::  MUMBLE_ENVIRONMENT_STORE     - Path to the folder the build environment is stored in.
-::  MUMBLE_ENVIRONMENT_SOURCE    - Build environment web source folder URL.
-::  MUMBLE_ENVIRONMENT_TRIPLET   - vcpkg triplet.
-::  MUMBLE_ENVIRONMENT_VERSION   - Full build environment version
-::                                 Must match archive and extracted folder name.
+::  MUMBLE_ENVIRONMENT_PATH      - Path to the environment's root directory.
 ::  MUMBLE_ENVIRONMENT_TOOLCHAIN - Path to the vcpkg CMake toolchain, used for CMake.
+::  MUMBLE_ENVIRONMENT_TRIPLET   - vcpkg triplet.
 ::  VCVARS_PATH                  - Path to the Visual Studio environment initialization script.
+::
+:: Defined on Azure Pipelines:
+::
+::  BUILD_NUMBER_TOKEN           - Access token for the build number page on our server.
+::
 
 @echo on
 
-for /f "tokens=* USEBACKQ" %%g in (`python "scripts/mumble-version.py"`) do (set "VER=%%g")
+if defined BUILD_NUMBER_TOKEN (
+	for /f "tokens=* USEBACKQ" %%g in (`python "scripts/mumble-version.py" --project`) do (set "VERSION=%%g")
+	for /f "tokens=* USEBACKQ" %%g in (`curl "https://mumble.info/get-build-number?version=%VERSION%&token=%BUILD_NUMBER_TOKEN%"`) do (set "BUILD_NUMBER=%%g")
```

Comment:
What does this do? An explanatory comment would probably be helpful here...

Reply:
Batch magic!

https://stackoverflow.com/a/6362922

Reply:
Holy shit is that ugly xD
We definitely need the link you just posted as a comment in the script as this is definitely something that can cause confusion ^^

Reply:
I strongly agree.

Reply:
Imo the comment should be really right there and not buried in the script's header :point_up: